### ISSUE TITLE
Add AccessKey's create-date to the User's filter table

### ISSUE THREAD

**Issue by chris-redekop** [chef/inspec-aws#64]
In #37, we identify an existing access-key field, its create date, to implement CIS recommendation 1.4.  Before we can search for access keys by this field, however, it needs to be added to the user's filter table.

**Comment by clintoncwolfe** [comment]
I really wish we could just loop over all access keys (and I'm looking hard at the discussion around the credential report on #38) - the fact that we have to fetch them user-by-user is an implementation detail, that I'd like to hide from the user.  To say, "does this account have any keys that have not been used in the last 90 days?" you'd want to say:

```
describe aws_iam_access_keys.where { |k| k.last_used_date < Time.now - 90 * 86400 } do
  it { should_not exist }
end
```

Assuming we don't use credentials report, we would implement aws_iam_access_keys to (internally) fetch all users, then loop over all users and fetch their 0-2 access keys, building a table from that.  That sounds slow, but we have to do the same thing if we implement it in the user resource - but then we place the burden of looping on the control author.

**Comment by chris-redekop** [comment]
I prefer your alternative, Clinton.  And I agree that it would be the same work, whether we put it behind `aws_iam_users` or `aws_iam_access_keys`.

I think I avoided this early on because plural/singular resources were new and scary--we were just getting started on users.

**Comment by clintoncwolfe** [comment]
Nod.  I've started on an implementation of a plural aws_iam_access_keys, which fetches all users and then fetches each key under the hood.

**Comment by chris-redekop** [comment]
Fixed by #112.
